src/v3/views/MidweekMeeting_PDF/MidweekMeetingPartsTitle.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Text, View, StyleSheet, Font } from '@react-pdf/renderer';
+import InterSemiBold from '@assets/fonts/Inter-SemiBold.ttf';
+import InterMedium from '@assets/fonts/Inter-Medium.ttf';
+import { MeetingPartsTitleProps } from './midweekMeeting.types';
+
+Font.register({
+  family: 'Inter',
+  fonts: [
+    { src: InterMedium, fontWeight: 'medium' },
+    { src: InterSemiBold, fontWeight: 'semibold' },
+  ],
+});
+
+const MidweekMeetingPartsTitle = ({ part, color, icon, taskConductor }: MeetingPartsTitleProps) => {
+  const styles = StyleSheet.create({
```

Comment:
@IlkaA1: it would be better to define these styles outside of the react component. All dynamic styles can be applied in the element directly. https://react-pdf.org/repl?example=mixed-styles